src/icons/ControlledAccessIcon.js

```diff
@@ -10,7 +10,7 @@ export default ({ width = 15, height = 15, fill = '#a9adc0', className = '', ...
     height={height}
   >
     <defs />
-    <title>icon-controlled-access</title>
+    { props.hideTitle ? null : <title>icon-controlled-access</title> }
     <path
       fill={fill}
       class="cls-1"
```

Comment:
`className` instead of `class` React will complain :)

Reply:
Also fixing other files with this problem.